react-fa/index.d.ts

```diff
@@ -1,20 +1,22 @@
-// Type definitions for react-fa v5.0.0
+// Type definitions for react-fa 4.1
```

Comment:
I was surprised to see the version going backwards, but `react-fa@5.0` was never published on NPM, so maybe @patsissons in #12655 updated it thinking that it reflected the *type* version and not the *package* version.